google-cloud-bigquery/src/main/java/com/google/cloud/bigquery/ViewDefinition.java

```diff
@@ -44,7 +44,7 @@
 
     private String query;
     private List<UserDefinedFunction> userDefinedFunctions;
-    private Boolean useLegacySql;
+    private Boolean useLegacySql = false;
```

Comment:
It seems like `null` is supported (handled in `useLegacySql()` method properly and allowed on `setUseLegacySql()` method). In such cases there is no need to set this to `false`, default `null` value should be fine and is cleaner.

This comment is applicable only if the other one (Line 204) remains unaddressed (otherwise this must be set to `false`, because interpretation of `null` will be changed).